File: app/src/main/java/com/babestudios/companyinfouk/data/model/officers/Address.java
package com.babestudios.companyinfouk.data.model.officers;


import com.google.gson.annotations.SerializedName;

public class Address {
	@SerializedName("locality")
	public String locality;
	@SerializedName("premises")
	public String premises;
	@SerializedName("postal_code")
	public String postalCode;
	@SerializedName("country")
	public String country;
	@SerializedName("address_line_1")
	public String addressLine1;
	@SerializedName("region")
	public String region;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/officers/Address.kt
package com.babestudios.companyinfouk.data.model.officers


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Address(
		@SerializedName("country")
		var country: String? = null,
		@SerializedName("postal_code")
		var postalCode: String? = null,
		@SerializedName("address_line_1")
		var addressLine1: String? = null,
		@SerializedName("locality")
		var locality: String? = null,
		@SerializedName("region")
		var region: String? = null,
		@SerializedName("premises")
		var premises: String? = null
) : Parcelable
